cmd/tailscaled/tailscaled.go

```diff
@@ -379,6 +387,12 @@ func tryEngine(logf logger.Logf, linkMon *monitor.Mon, name string) (e wgengine.
 		ListenPort:  args.port,
 		LinkMonitor: linkMon,
 	}
+	if args.birdSocketPath != "" && createBIRDClient != nil {
+		conf.BIRDClient, err = createBIRDClient(args.birdSocketPath)
```

Comment:
do you want to do some logging at least before this line, as createBIRDClient can block (potentially forever) if bird doesn't reply with its greeting banner.

```go
    log.Printf("Connecting to bird at %s ...", args.birdSocketPath)
    conf.BIRDClient, err = createBIRDClient(args.birdSocketPath)
```